rust/agama-lib/src/utils/file_format.rs

```diff
@@ -98,7 +98,10 @@ impl FileFormat {
     ///
     /// It returns `true` if the content starts with a shebang.
     fn is_script(content: &str) -> bool {
-        content.starts_with("#!")
+        let Some(first_line) = content.lines().next() else {
+            return false;
+        };
+        first_line.starts_with("#!") && !first_line.contains("jsonnet")
```

Comment:
we should add small docu line explaining why we cannot handle jsonnet as others ( we need injection and also it produce json unlike other scripts which is standalone and using agama CLI )